index.js

```diff
@@ -0,0 +1,2 @@
+require('./format-as-currency');
```

Comment:
ah, makes sense. try pulling again from the umd branch.

it's a bit strange since the module name is "bcherny/formatAsCurrency", not "format-as-currency". i'm not sure how the latter works for you.